Tools/LyTestTools/ly_test_tools/launchers/platforms/base.py

```diff
@@ -255,7 +255,8 @@ def wait(self, timeout=30):
         """
         ly_test_tools.environment.waiter.wait_for(
             lambda: not self.is_alive(),
-            exc=ly_test_tools.launchers.exceptions.WaitTimeoutError("Application is unexpectedly still active"),
+            exc=ly_test_tools.launchers.exceptions.WaitTimeoutError(f"Application is unexpectedly still active after "
+                                                                    f"timeout of {timeout} seconds"),
```

Comment:
Not a comment on your changes, but on the structure of this function. It's simply called "wait" but it's clearly a "wait_for_exit"

A rename would further clarify the exception via stacktrace ...but may also be of heinous scope.